internals-js/src/tagSpec.ts

```diff
@@ -41,11 +41,11 @@ export class TagSpecDefinition extends FeatureDefinition {
       name:'tag',
       locations: this.tagLocations,
       repeatable: true,
-      argumentFct: (schema) => ({
-        args: [{ name: 'name', type: new NonNullType(schema.stringType()) }],
-        errors: [],
-      }),
+      args: [{ name: 'name', type: (schema) => new NonNullType(schema.stringType()) }],
+      composes: true,
+      supergraphSpecification: () => TAG_VERSIONS.latest(),
```

Comment:
No, it's a good point, though I'm just perpetuating [the status quo](https://github.com/apollographql/federation/blob/main/composition-js/src/merging/merge.ts#L228-L231) here. But I agree we should probably change that behaviour and use the smallest version that is needed instead of always defaulting to the last.

I can do that change here if we really want to, but I feel it's related to using the latest `join` spec in the supergraph which I've been meaning to open a ticket for. So I've created #2495 and I would implement this change of strategy there.